styx-flyte-client/src/main/java/com/spotify/styx/flyte/client/FlyteAdminClient.java

```diff
@@ -73,20 +95,43 @@ public static FlyteAdminClient create(
     final var insecure = config.getBoolean(FLYTEADMIN_INSECURE);
     final var grpcDeadlineSeconds = config.getLong(FLYTEADMIN_GRPC_DEADLINE_SECONDS);
     final var maxRetryAttempts = config.getInt(FLYTEADMIN_MAX_RETRY_ATTEMPTS);
+    final var initialIntervalMillis = config.hasPath(FLYTEADMIN_RETRY_INTERVAL_INITIAL_MILLIS) ? config.getInt(FLYTEADMIN_RETRY_INTERVAL_INITIAL_MILLIS): DEFAULT_FLYTEADMIN_RETRY_INTERVAL_INITIAL_MILLIS;
+    final var multiplier = config.hasPath(FLYTEADMIN_RETRY_INTERVAL_BACKOFF_MULTIPLIER)? config.getDouble(FLYTEADMIN_RETRY_INTERVAL_BACKOFF_MULTIPLIER): DEFAULT_FLYTEADMIN_RETRY_INTERVAL_BACKOFF_MULTIPLIER;
+
+    final RetryConfig retryConfig = getRetryConfig(maxRetryAttempts, initialIntervalMillis, multiplier);
 
     var builder = ManagedChannelBuilder.forTarget(target);
 
     if (insecure) {
       builder.usePlaintext();
     }
     builder.intercept(GrpcClientMetadataInterceptor.create(serviceName));
-    // Enable transparent retries:
-    // https://github.com/grpc/proposal/blob/master/A6-client-retries.md#transparent-retries
     var channel =
-        builder.enableRetry().maxRetryAttempts(maxRetryAttempts).intercept(interceptors).build();
+        builder.enableRetry().maxRetryAttempts(0).intercept(interceptors).build();
```

Comment:
Does `maxRetryAttempts(0)` mean retry forever?

Reply:
@honnix  can you explain what this does?

```
RPC failures can occur in four distinct ways:

1.The RPC fails at the client-side load balancing step.
2.Load balancing succeeds but the RPC never leaves the client.
3.The RPC reaches the server, but has never been seen by the server application logic.
4.The RPC is seen by the server application logic, and fails.
```

Number 4 is introduced by this PR and handled by resilience4j.
2-3 is enabled by having `enableRetry().maxRetryAttempts(0)` as far as we can tell.

Reply:
`enableRetry().maxRetryAttempts(0)` is to explicitly ensure we have transparent retry but not hedging policy which we don't have.

1 is a bit unclear but I guess it can also be handled by hedging policy. In our case, it will just fail the RPC immediately.